course_discovery/apps/api/tests/test_serializers.py

```diff
@@ -433,6 +445,17 @@ def test_include_retired_programs(self):
             NestedProgramSerializer([retired_program], many=True, context=self.serializer_context).data
         )
 
+    def test_filter_course_runs_on_edx_org_short_name(self):
+        """
+        Verify courses runs filtering on edX organization.
+        """
+        edx_org_short_name = 'edx'
+        edx_org_course = CourseRunFactory(course__authoring_organizations__key=edx_org_short_name)
+        serializer = CourseRunWithProgramsSerializer.prefetch_queryset(
```

Comment:
this is not a serializer, its serialized response, right?
`serializer` -> `serialized_course_runs`

Reply:
Done